src/services/response-service.ts

```diff
@@ -28,8 +31,7 @@ export default class ResponseService extends PercyClientService {
     }
 
     // Capture if the resourceUrl has a hostname in the allowedHostnames
-    const parsedResourceUrl = new URL(resourceUrl)
-    if (this.allowedHostnames.some((hostname) => parsedResourceUrl.hostname === hostname)) {
+    if (this.allowedHostnames.some((hostname) => domainMatch(hostname, resourceUrl))) {
```

Comment:
The port number in the hostname causes domainMatch to fail. I just need to add back the `new URL` bit after lunch and this should work.